src/app/pages/global-reach/map/map.js

```diff
@@ -428,19 +450,31 @@ export default class Map1 extends Controller {
         }
     }
     makeFitBoundsArray(dataArray) {
-        const latlngArray = dataArray.map((e) => {
-            return [Number.parseFloat(e.fields.lat), Number.parseFloat(e.fields.long)];
+        let latlngArray = dataArray.map((e) => {
+            return [Number.parseFloat(e.fields.long), Number.parseFloat(e.fields.lat)];
         });
 
+        latlngArray = latlngArray.filter((i) => (i[0] !== 0 && i[1] !== 0));
```

Comment:
Is this logic correct, or should it be or? This filters out if either one is zero. Also, you could chain this onto the previous statement (and have it go back to `const`).

Reply:
We want to filter out if both are zero and keep the rest. I chained this and moved it back to constant.

Reply:
Here you are _including_ where _both_ are non-zero, so filtering out where at least one is zero. If you change `&&` to `||`, you will include with at least one is non-zero and only filter out where both are zero.